.eslintrc.json

```diff
@@ -8,6 +8,23 @@
 		"curly": "off",
 		"indent": "off",
 		"no-confusing-arrow": "off",
-		"no-mixed-operators": "off"
+		"no-mixed-operators": "off",
+		"import/no-extraneous-dependencies": [
+			"error", {
+				"devDependencies": true,
+				"packageDir": "./"
```

Comment:
Why is this necessary to specify?

Reply:
eslint should read the dependencies from parent `package.json` instead of the `package.json` in each module.